integration_tests/test_parser.py

```diff
@@ -15,64 +15,76 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import os
+import fixtures
 import subprocess
 
 import integration_tests
+import testscenarios
 from snapcraft.tests import fixture_setup
 
 
-class TestParser(integration_tests.TestCase):
+class ParserTestCase(integration_tests.TestCase):
     """Test bin/snapcraft-parser"""
 
-    def test_parser_basic(self):
-        """Test snapcraft-parser basic usage"""
-        fixture = fixture_setup.FakePartsWiki()
-        self.useFixture(fixture)
+    def setUp(self):
+        super().setUp()
 
-        args = [self.snapcraft_parser_command, '--index',
-                fixture.fake_parts_wiki_fixture.url,
-                '--debug', '--output', 'parts.yaml']
-        subprocess.check_call(args, stderr=subprocess.DEVNULL,
-                              stdout=subprocess.DEVNULL)
+        tempdir = fixtures.TempDir()
+        self.useFixture(tempdir)
+        self.useFixture(fixtures.EnvironmentVariable('TMPDIR', tempdir.path))
+        self.tempdir = tempdir.path
```

Comment:
The cwd set by the parent test class is already a temp dir, and you can access it with self.path. Do we need another dir?